src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +71,11 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    /**
+     * Returns true when command mutates data
+     */
+    public boolean isMutating() {
+        return COMMAND_NOT_MUTATING;
```

Comment:
As mentioned during tutorial, we shouldn't assign a default behavior to the parent class even though its generally true/false
E.g. what if we assign `canFly()` to be true by default and we forgot to override it for some animals such as penguin?
result -> suddenly penguin can fly